src/main/java/com/minecolonies/compatibility/tinkers/SlimeTreeCheck.java

```diff
@@ -9,8 +9,7 @@
 /**
  * This class is to store a check to see if a tree is a slime tree.
  */
-public final class SlimeTreeCheck extends SlimeTreeProxy
-{
+public final class SlimeTreeCheck extends SlimeTreeProxy {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)